python/lsst/cp/pipe/ptc.py

```diff
@@ -684,6 +680,9 @@ def measureMeanVarCov(self, exposure1, exposure2, region=None, covAstierRealSpac
         wDiff = np.where(diffIm.getMask().getArray() == 0, 1, 0)
         w = w12*wDiff
 
+        if np.sum(w) == 0:
+            return np.nan, np.nan, None
```

Comment:
Why doesn't this return `np.nan, np.nan, np.nan`?  The docstring says that's the failure mode.  Should this return an empty list instead?

Reply:
This is the same failure mode that we have a few lines above when {mu1} or {mu2} ara NaN. I will update the docstring.